src/Shared/Im/View/ChatInput.purs

```diff
@@ -127,15 +126,14 @@ chatBarInput
                                     <>
                                           [ HA.class' "chat-input"
                                           , HA.id $ show elementId
-                                          , HA.placeholder $ if isWebSocketConnected then "Type here to message " <> recipientName else "Waiting for connection..."
-                                          , HA.disabled $ not isWebSocketConnected
-                                          , SK.keyDownOn "Enter" EnterBeforeSendMessage
+                                          , HA.placeholder $ if isWebSocketConnected then "Type here to message " <> recipientName else "Can't send a message while there is no connection"
```

Comment:
this is also too long for mobile, and it gives the impression you cant type at all, how about `reconnecting...` or something like that